src/Sulu/Bundle/WebsiteBundle/Routing/ContentRouteProvider.php

```diff
@@ -148,7 +148,11 @@ public function getRouteCollectionForRequest(Request $request)
                 return $collection;
             }
 
-            if (preg_match('/\/$/', $resourceLocator) && $prefix) {
+            if (preg_match('/\/$/', $resourceLocator) && ('/' !== $resourceLocator || $prefix)) {
+                if (is_null($prefix)) {
```

Comment:
But I think you don't need that check at all, because null would be casted to an empty string anyway. Or was there a reason for that?

Reply:
@danrot fixed